scripts/install_third_party.py

```diff
@@ -443,6 +444,12 @@ def download_and_untar_files(
 SIMPLEJSON_TAR_ROOT_NAME = 'simplejson-%s' % SIMPLEJSON_REV
 SIMPLEJSON_TARGET_ROOT_NAME = SIMPLEJSON_ROOT_NAME
 
+NUMPY_REV = '1.6.1'
+NUMPY_ROOT_NAME = 'numpy-%s' % NUMPY_REV
+NUMPY_TAR_URL = "http://downloads.sourceforge.net/project/numpy/NumPy/%s/numpy-%s.tar.gz" % (NUMPY_REV, NUMPY_REV)
```

Comment:
Please wrap to 80 chars, following the examples above (SIMPLEJSON_TAR_URL, etc.)